t/include_conf_dir/include_conf_dir.t.in

```diff
@@ -0,0 +1,84 @@
+#!@PERL@
```

Comment:
Test setup is quite complicated. Quite hard to follow. I'm against dynamically generating new files/folders for a given test. Should be able to use what's already there. If not just add a new static test dir or file. can even backup your own test config file.